lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Lexik\\Bundle\\JWTAuthenticationBundle\\LexikJWTAuthenticationBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%"
+    },
+    "env": {
+        "JWT_PRIVATE_KEY_PATH": "%kernel.project_dir%/var/jwt/private.pem",
```

Comment:
I'm not sure if this will work ... but there's no `%VAR_DIR%` value (only ETC, CONF, BIN, SRC and WEB). I'm going to ask about this in the Flex repository.